BeehiveCommunity/Frontend_Mentor
# Add Pricing Component with Toggle

## What does this PR do?

This pull request implements a new pricing component with a toggle feature that allows users to switch between monthly and yearly pricing options. The component enhances user experience by providing a straightforward comparison of pricing plans.

Fixes #656 

![Screenshot 2024-11-01 210406](https://github.com/user-attachments/assets/014c4ec5-eb55-4077-b080-7f60bb5992ca)

## Type of change



- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## How should this be tested?

1. Pull this branch.
2. Open the project in a browser.
3. Navigate to the pricing component(33-pricing-component-with-toggle).
4. Toggle between monthly and yearly to verify that the pricing updates correctly.

## Mandatory Tasks
